plugins/rplanners/ParabolicPathSmooth/ppramp.cpp

```diff
@@ -123,7 +123,7 @@ bool PPRamp::SolveMinTime2(Real amax,Real timeLowerBound)
     }
     _a2 = -_a1;
     PARABOLIC_RAMP_ASSERT(ttotal >= timeLowerBound);
-    Real ts1,ts2;
+    Real ts1 = 0.0, ts2 = 0.0;
```

Comment:
In this context, I think it's better to initialize variables that represent time instants with `-1.0` rather than `0.0` to indicate that they are invalid.

Reply:
changed to -1